docs/howto/operator/define_extra_link.rst

```diff
@@ -0,0 +1,53 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This is not a good place to put this information.  This section should contain information on how to use and not how to create operators

Reply:
I feel like it make sense to put all operator related docs all under one section, e.g. `docs/howto/operator` so that it is easier to users to find the related information.

Reply:
I think the doc has been reconstructed to put all the user phasing info. 

@ashb , I think we need to define or provide clearer guideline on which doc should be put at which folder.  Maybe through the season of doc initiative or an AIP.

Reply:
@milton0825 It makes sense, but only all the information about the operators would be in one place. This is not true in this case. I will suggest a different structure because I am concerned that the number of guides on the use of operators will grow rapidly. In this case, the guides about creating a new operator can be easily overlooked when it is just a small sub-point in large list.

I would like these fragment in a new page that will contain information on the creation of operators.  As a part of SOD 2009, I proposed the project "4. How to create a workflow" and one of its expected deliverables is to create such a page.
> A page for developing custom operators that includes:
> * Describing mechanisms that are important when creating an operator, such as > template fields, UI color, hooks, connection, etc.
> * Describing the responsibility between the operator and the hook
> * Considerations for dealing with shared resources (such as connections and hooks)

https://cwiki.apache.org/confluence/display/AIRFLOW/Season+of+Docs+2019

As mentors of SOD project, I set a personal goal to improve the structure of the documentation.  I agree that the recommendations will be useful and I will try to develop them.

I hope my explanations were helpful.